Source/CurrencyCloud/Client.cs

```diff
@@ -493,6 +493,20 @@ public async Task<MarginBalanceTopUp> TopUpMarginBalanceAsync(string currency, d
 
         #region Beneficiaries
 
+        /// <summary>
+        /// Verifies account details
+        /// </summary>
+        /// <param name="beneficiaryAccountVerification">Account data to be verified</param>
+        /// <returns>Asynchronous task, which returns the account verification response.</returns>
+        /// <exception cref="InvalidOperationException">Thrown when client is not initialized.</exception>
+        /// <exception cref="ApiException">Thrown when API call fails.</exception>
+        public async Task<BeneficiaryAccountVerification> VerifyAccountAsync(BeneficiaryAccountVerificationRequest beneficiaryAccountVerification)
```

Comment:
BeneficiaryAccountVerification and BeneficiaryAccountVerificationRequest seem to be missing